main/main.py

```diff
@@ -1077,7 +1085,7 @@ def complement_bgr(color):
                     next_frame_path_list = get_next_frame_path_list(video_name, img_path)
                     # initial frame
                     init_frame = img.copy()
-                    label_tracker = LabelTracker('KCF', init_frame, next_frame_path_list) # TODO: replace 'KCF' by 'CSRT'
+                    label_tracker = LabelTracker('DASIAMRPN', init_frame, next_frame_path_list) # TODO: replace 'KCF' by 'CSRT'
```

Comment:
looks like a hardcoded change of default?

Reply:
Yes. It seemed to me that the old way of changing which algo you want to use was to go into the source code and hard code a change so that's what I did here. There should probably be an argument added to the CLI for this.

Reply:
Yeah, we should turn it into an argument via `argparse`. You could make the `DASIAMRPN` the default one.

Reply:
I'm not sure it should be default as I am not sure how it deals with not running on a GPU. I will add functionality with argparse for selecting which algorithm you want to use however.

Reply:
True, we should keep one of the OpenCV simple ones as the default one.